cmd/multiFaExtract/multiFaExtract.go

```diff
@@ -15,30 +15,35 @@ func multiFaExtract(s Settings) {
 	var ans []fasta.Fasta
 	records := fasta.Read(s.InFile)
 	if s.Bed == "" {
-		if !(s.Start < s.End) {
-			log.Fatalf("Invalid arguments, start must be lower than end")
-		}
-		ans = extractMultiHelper(records, s.Start, s.End)
-		if s.RemoveGaps {
-			ans = fasta.RemoveGaps(ans)
-		}
+		ans = extractMultiHelper(records, s.Start, s.End, s.RemoveGaps, false)
 		fasta.Write(s.OutFile, ans)
 	} else {
 		bedChan := bed.GoReadToChan(s.Bed)
 		for b := range bedChan {
-			ans = extractMultiHelper(records, b.ChromStart, b.ChromEnd)
-			if s.RemoveGaps {
-				ans = fasta.RemoveGaps(ans)
+			ans = extractMultiHelper(records, b.ChromStart, b.ChromEnd, s.RemoveGaps, b.Strand == bed.Negative)
+			if b.FieldsInitialized >= 4 {
+				fasta.Write(fmt.Sprintf("%s.fa", b.Name), ans)
```

Comment:
Can you help me understand why we would name the output file differently if the fields initialized are less than or equal to 4? I'm assuming it would be to not lose information, but then we do not seem to include the chrom start end anymore?